.github/workflows/audit.yml

```diff
@@ -23,15 +23,6 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
-      # Actions-rs/install is unmaintained but works for now.
-      # Alternatives for when it breaks:
-      #  - dtolnay/install - not a full replacement - only supports crates he uses (not audit currently)
-      #  - baptiste0928/cargo-install - looks like it just compiles the crate and then caches it for subsequent runs
-      #  - just `cargo install` and caching it
-      - uses: actions-rs/install@v0.1
```

Comment:
This change seems to remove installation of `cargo-audit` entirely?

Reply:
That's correct.

Reply:
It appears to be unnecessary. Maybe the GitHub runners come with it?